### PR TITLE
docs: add vercel logo

### PR DESCRIPTION
**Pull request by posthardcode**
## Purpose

Adds `Powered by Vercel` logo as required for OSS sponsorship.

## Approach

Logo was added to README only, as Storybook doesn't allow you to customize beyond the brand logo.

## Testing

N/A

## Risks

N/A

### CODE REVIEW

**Comment by josokinas** [comment]
Maybe better to put this underneath Storybook banner, @rabelloo what do you think?

**Review by josokinas — APPROVED** [review]

**Comment by josokinas** [comment]
Removing a link to https://github.com/onfido/castor/issues/159, as it would close the issue same as https://github.com/onfido/castor-icons/pull/200 did.

File: README.md [hunk 1]
```diff
@@ -27,3 +27,5 @@ npm install @onfido/castor-react
 ```
 
 Also setup following [React documentation](./packages/react).
+
+[![Vercel](https://www.datocms-assets.com/31049/1618983297-powered-by-vercel.svg)](https://vercel.com/?utm_source=onfido-oss&utm_campaign=oss)
```
[the remark this answers is not in the document]
Comment (rabelloo) [reply README.md:31]: At the very bottom would be better than close to the top badges, Vercel's is too big and would draw too much attention over Storybook.

I like yours, though I'd rename the section slightly.

```suggestion
## Sponsors

[![Vercel](https://www.datocms-assets.com/31049/1618983297-powered-by-vercel.svg)](https://vercel.com/?utm_source=onfido-oss&utm_campaign=oss)
```

[the remark this answers is not in the document]
Comment (josokinas) [reply README.md:31]: Yeah, I'm fine with "sponsors" 👍

File: README.md [hunk 2]
```diff
@@ -27,3 +27,6 @@ npm install @onfido/castor-react
 ```
 
 Also setup following [React documentation](./packages/react).
+## Sponsors
```
Comment (josokinas) [inline README.md:30]: Sorry, missed an empty line here:

```suggestion
Also setup following [React documentation](./packages/react).

## Sponsors
```

**Comment by posthardcode** [comment]
I've accepted the review directly here and the commit doesn't pass linting; amending it and force pushing.

**Comment by josokinas** [comment]
Yes, this is a known issue.

We do require a conventional commit standard, however GitHub commit suggestions do not support that.

Usually what I do - I edit the commit before accepting a change, e.g. `style: accept suggested changes`.

But another option is to force-push, which is a little more annoying however we do allow force-pushing on feature branches.

**Pull request merged**